src/core/Akka.MultiNodeTestRunner/Discovery.cs

```diff
@@ -108,14 +108,8 @@ private IEnumerable<RoleName> RoleNames(Type specType)
             var ctorWithConfig = FindConfigConstructor(specType);
             var configType = ctorWithConfig.GetParameters().First().ParameterType;
             var args = ConfigConstructorParamValues(configType);
-            var configInstance = Activator.CreateInstance(configType, args);
-            var roleType = typeof(RoleName);
-            var configProps = configType.GetProperties(BindingFlags.Instance | BindingFlags.Public);
```

Comment:
Don't believe we need the property checking anymore using reflection - a full MNTR run (which this PR will trigger) will show up with many fewer tests and much lower execution time if this change made several tests suddenly stop showing up.